pkg/cdi/container-edits_unix.go

```diff
@@ -20,11 +20,42 @@
 package cdi
 
 import (
+	"errors"
 	"fmt"
 
-	runc "github.com/opencontainers/runc/libcontainer/devices"
+	"golang.org/x/sys/unix"
 )
 
+const (
+	blockDevice = "b"
+	charDevice  = "c" // or 'u'
```

Comment:
nit: Should strictly be to match the `string` type:
```suggestion
	charDevice  = "c" // or "u"
```

Reply:
Let's update it for consistency if you don't mind.